benchmarks/cluster3nodes/configuration/config-node-explorer.yaml

```diff
@@ -0,0 +1,174 @@
+# global filter; messages must have at least this severity to pass:
```

Comment:
this configuration is not necessary. please, remove it.

Reply:
Done.

Reply:
Done.